src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,48 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.addAlive(this);
+  }
+
+  static alive = [];
```

Comment:
The use of a static field `alive` is correct, but ensure that this field is properly managed when animals are created or removed to avoid memory leaks or incorrect state.